index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="first-navbar">
+            <a href="https://www.facebook.com"> <img src="./images/facebook_icon.png"> </a>
+            <a href="https://www.facebook.com"> <img src="./images/twitter_icon.png"> </a>
+        </div>
+        <nav>
+            <div class="logo"> <a href="index.html"></a>
+            </div>
+            <div class="nav-bar">
+                <span class="hamburger-icon1">&#9776</span>
+                <span> <a href="index.html"> <img src="./images/logo.jpg" alt=""  class="header-logo" srcset=""></a></span>
+                <span class="cross hidden">X</span>
+            <ul class="second-navbar">
+                <li class="second-navbar-links">About</li>
+                <li class="second-navbar-links"><a href="#"></a> Products</li>
+                <li class="second-navbar-links"><a href="#"></a> News</li>
+                <li class="second-navbar-links"><a href="#"></a> login</li>
+            </ul>
+        </div>
```

Comment:
I see that you have not implemented the Navbar for the desktop view of the website. Notice in the `your work` screenshot below, on a screen size of `1375`, it is still showing the mobile version. Please implement the desktop version.

The navbar of the desktop version should be as follows. 
- It is divided in a top dark bar and a bottom white bar.
- The top short bar section with social media links, logout, and language links aligned to the right.
- The taller bottom bar with a logo and a list of page links.
- The top bar contains a logo to the left.
- The top bar contains a list of page links aligned to the right.

Your work 
![image](https://github.com/alizaamir123/Capstone-project/assets/84156714/4dac55ce-c635-43f3-996f-9a9eeecf57ed)

Design template
![image](https://github.com/alizaamir123/Capstone-project/assets/84156714/bf64bea8-f7a6-4c1b-9d8d-cb5f703032c9)